core/misc/material/ICON-LICENSE.txt

```diff
@@ -0,0 +1,201 @@
+                                Apache License
```

Comment:
Apache 2 isn't a license we can include in core. 

Is this for the plus and minus icons? We're already using font-awesome icons elsewhere in core. Can we just use those for these icons too?

Reply:
Actually the Apache license seems to already be in core: https://github.com/backdrop/backdrop/blob/1.x/core/misc/opensans/FONT-LICENSE.txt.

However, I'm fine with using font-awesome icons, I just didn't see any in core when I first looked.